msteams-platform/bots/how-to/teams-conversational-ai/how-conversation-ai-get-started.md

```diff
@@ -783,6 +783,86 @@ A feedback loop helps monitor and improve the bot’s interactions, leading to m
 * **Learn from Mistakes**: The model learns from correct behavior examples to avoid similar mistakes in the future.
 * **Handle Complex Commands**: The model becomes capable of handling more complex commands after learning from its mistakes.
 
+## Upgrade your conventional bot to custom engine agent
+
+If you already have a bot on Teams, you can upgrade your existing bots to custom engine agent that support streaming, citations, and AI labels. This is required for the agent to align with the conversational AI UX paradigm and provide a consistent experience with declarative agents.
+
+> [!NOTE]
+>
+> * Custom engine agent support for Microsoft 365 Copilot Chat is available only in [public developer preview](../../../resources/dev-preview/developer-preview-intro.md).
+> * Custom engine agent isn't supported in Python.
+
+Here is the required to-do list to upgrade your bot to custom engine agent:
+
+| To-Do List | Supporting docs |
+| ---- | ---- |
+| Update the AI SDK versions |  • If you are using JavaScript, update the version to [v1.6.1](https://www.npmjs.com/package/@microsoft/teams-ai).  <br> • If you are using Csharp, update the version to [v1.8.1](https://www.nuget.org/packages/Microsoft.Teams.AI/1.8.1). |
+| Enable streaming for bot. | [Stream bot messages](../../streaming-ux.md) |
+| Use AI labels to identify that the message was generated using AI. | [AI labels](../bot-messages-ai-generated-content.md#ai-label)|
+| Use citations to refer to the source of the bot message through in-text citations and references.| [Citations](../bot-messages-ai-generated-content.md#citations)|
+
+## Add support for Microsoft 365 Copilot Chat
+
+You can now add support for custom engine agents in Microsoft 365 Copilot Chat, which is currently in public developer preview. Additionally, you can preview support for asynchronous patterns, including follow-up messages and long-running tasks. For more information, see [asynchronous patterns](/microsoft-365-copilot/extensibility/ux-custom-engine-agent).
+
+If you want your custom engine agent to support Microsoft 365 Copilot Chat, add the following into your app manifest:
+
+1. Add the app manifest property `copilotAgents` and its sub property `customEngineAgents` into your [app manifest](../../../resources/schema/manifest-schema-dev-preview.md#customengineagents):
+
+    ```json
+    "copilotAgents": { 
+        "customEngineAgents": [ 
+          { 
+            "type": "bot", 
+            "id": "<Bot-Id-Guid>" 
+          } 
+        ] 
+      }
+    
+    ```
+
+1. In your app manifest, set the `scopes` as `personal`:
```

Comment:
Confirm on this scope part as well, if it is supported for bot or commands.
Also, Bizchat is supported only in personal scope or other scopes as well?

Reply:
Updated as suggested